src/components/layout/Header/HeaderContacts/HeaderContacts.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * HeaderContacts component.
+ * @module components/ItaliaTheme/Header/HeaderContacts/HeaderContacts
+ */
+
+import React from 'react';
+import { UniversalLink } from '@plone/volto/components';
+import { Icon } from 'io-sanita-theme/components';
+
+import './headerContacts.scss';
+
+const HeaderContacts = () => {
+  //esempio items - arriverà via props
+  const items = [
+    { text: 'testo 1', href: '0239238348', href_type: 'tel' },
```

Comment:
occhio che qui non avrai mai href e href_type, avrai solo href con dentro scritto una url che può essere del tipo.

- http://www.sitoweb.it
- https://www.sitoweb.it
- tel:0532123456
- mailto:teto@prova.it

Reply:
Certo, da quello che ho capito, UniversalLink gestisce questa cosa, certo? Quindi basta href, quindi dovrei trovare un modo per scegliere l'icona basato su questo

Reply:
No, sbaglio io, UniversalLink non gestisce "tel:/mailto:" ma riesce a capire un numero come telefono. Provo a verificare